Packages/com.unity.inputsystem/InputSystem/Plugins/InputForUI/InputSystemProvider.cs

```diff
@@ -127,7 +127,7 @@ public void Update()
             // This is necessary to ensure that events are dispatched in the correct order.
             // If all events are of the PointerEvents type, sorting is based on reverse order of the EventSource enum.
             // Touch -> Pen -> Mouse.
-            m_Events.Sort(SortEvents);
+            m_Events.Sort((a, b) => SortEvents(a, b));
```

Comment:
So using a lambda expression here doesn't allocate like mentioned in the forums? Interesting learning!